src/diffusers/loaders/textual_inversion.py

```diff
@@ -481,11 +483,28 @@ def unload_textual_inversion(
 
         # Remove just one token
         pipeline.unload_textual_inversion("<moe-bius>")
+
+        # Examples 3 (based on SDXL LoRA Advanced training)
+        pipeline = AutoPipelineForText2Image.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0")
+        embedding_path = hf_hub_download(repo_id="LinoyTsaban/web_y2k_lora", filename="web_y2k_emb.safetensors", repo_type="model")
```

Comment:
```suggestion
        embedding_path = hf_hub_download(repo_id="linoyts/web_y2k", filename="web_y2k_emb.safetensors", repo_type="model")
```

I couldn't find that repo_id, I tested this one `linoyts/web_y2k` and it works  @linoytsaban